src/utils/safe-versions.ts

```diff
@@ -0,0 +1,40 @@
+import semverSatisfies from 'semver/functions/satisfies'
+
+import type { SafeInfo } from '@safe-global/safe-gateway-typescript-sdk'
+
+export enum VERSION_FEATURES {
+  SAFE_TX_GAS_OPTIONAL = 'SAFE_TX_GAS_OPTIONAL',
+  SAFE_TX_GUARDS = 'SAFE_TX_GUARDS',
+  SAFE_FALLBACK_HANDLER = 'SAFE_FALLBACK_HANDLER',
+  ETH_SIGN = 'ETH_SIGN',
+}
+
+const FEATURES_BY_VERSION: Record<VERSION_FEATURES, string> = {
+  [VERSION_FEATURES.ETH_SIGN]: '>=1.1.0',
+  [VERSION_FEATURES.SAFE_FALLBACK_HANDLER]: '>=1.1.1',
+  [VERSION_FEATURES.SAFE_TX_GAS_OPTIONAL]: '>=1.3.0',
```

Comment:
This doesn't seem to be used anywhere except tests.
I think it's supposed to be checked in the AdvancedParams?

Reply:
We already have a check in place there [based on the form values](https://github.com/safe-global/web-core/blob/dev/src/components/tx/AdvancedParams/AdvancedParamsForm.tsx#L110).

I don't think we should remove this from the `enum` though as we may need it at some point.

Reply:
I would remove it, you can leave it in the enum but comment out if you like, but we shouldn't keep code "for future".